Gem/Code/Source/Components/Multiplayer/WeaponEffectComponent.h

```diff
@@ -18,13 +18,7 @@ namespace MultiplayerSample
     public:
         AZ_MULTIPLAYER_COMPONENT(MultiplayerSample::WeaponEffectComponent, s_weaponEffectComponentConcreteUuid, MultiplayerSample::WeaponEffectComponentBase);
 
-        static void Reflect(AZ::ReflectContext* context);
-
-        static void GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required)
-        {
-            WeaponEffectComponentBase::GetRequiredServices(required);
```

Comment:
We shouldn't be putting code like this in headers in any case.. it's non-templated, it only gets called once at startup so there are no inlining concerns, but it does impact build times and code readability of the class definition. So, good it's deleted now I guess 👍